packages/esm-patient-vitals-app/src/vitals-and-biometrics-header/vitals-header.scss

```
@@ -11,7 +11,7 @@
   justify-content: space-between;
   align-items: center;
   min-height: layout.$spacing-09;
-  margin: layout.$spacing-03 0 0 layout.$spacing-05;
+  padding: layout.$spacing-03 0 0 layout.$spacing-05;
```

```suggestion
  padding: 0 0 0 layout.$spacing-05;
```

Thanks, @donaldkibet. We should probably remove the top margin altogether.